src/App.tsx

```diff
@@ -1,26 +1,52 @@
-import React from 'react';
-import logo from './logo.svg';
+import { useState, useEffect } from 'react';
 import './App.css';
+import './normalize.css'
+import { Wine } from './interfaces/wine_interface';
+import { WineList } from './components/WinesList/WineList'
+import { fetchAllWines } from './api/wine_api'
 
-function App() {
+const App = () => {
+  const [wines, setWines] = useState<Wine[]>([])
```

Comment:
Why do wines need to be stateful?

Reply:
This component fetches the data and stores it in state so it can then pass the state to its child component as a prop to render the list. Also if there are actions in child components that require the `wines` state to be updated, I can pass the `setWines` function as a callback to the child components.